libraries/GCS_MAVLink/GCS.h

```diff
@@ -129,6 +129,13 @@ class GCS_MAVLINK
     // call to reset the timeout window for entering the cli
     void reset_cli_timeout();
 
+    bool is_usb() { return chan == MAVLINK_COMM_0; }
```

Comment:
In linux `is_high_bandwidth` can return true for all of them :-)

baudrate isn't the be-all-and-end-all of high-bandwidth (921,600 but 80% packet loss!) - but it would certainly suck less than "are we on COMM_0".

My original `is_usb` was just putting into code what was in comments - it wouldn't have been accurate on Linux, of course.

I suggest we leave in as "is_high_bandwidth" for the time being and come back and do better later on the implementation - probably based, as you say, on baud rate.